docs/create-candy/01-introduction.md

```diff
@@ -44,7 +44,7 @@ The specific version numbers don't matter that much, but make sure you're runnin
 
 ### The Candy Machine Command Line Tool
 
-Creating and controlling a Candy Machine is typically done through command line tool, currently distributed as source code in a GitHub repository.
+Creating and controlling a Candy Machine is typically done through the Metaplex command line tool, currently distributed as source code in a GitHub repository.
```

Comment:
` typically done through command line tool` is missing an article (`a`/`the`), but was also a bit ambiguous since this page mentions many command line tools! I figured for clarity, it'd be best to specify that you mean the Metaplex CLI specifically.